### ISSUE TITLE
[Guns] Xray Gun

### ISSUE THREAD

**Issue by Swagile** [HippieStationCode/HippieStation13#3234]
If you spawn in a xray gun, they have no firing pin so you also have to manually spawn in a firing pin / box and load it in. This happened in a zombie round ages ago but I forgot to make a issue report, and a recent somewhat related issue came up that jogged my memory.

**Comment by ArcLumin** [comment]
This is intended, as a nerf to xray guns.

**Comment by ArcLumin** [comment]
admin spawns the actual item. Only reason you'd be able to spawn it with the pin in is if there was one coded to have the pin in too, AND for the admin to pick that one instead of the vanilla one

**Comment by KittyKathy** [comment]
Yeah there should be one that comes with a pin already that replaces the one in the centcom armory.
The laserguns in the thunderdome also spawn with no pins should fix that.
